lib/Runtime/Debug/TTEventLog.cpp

```diff
@@ -1256,7 +1262,14 @@ namespace TTD
         //Create the event object and add it to the log
         NSLogEvents::SnapshotEventLogEntry* snapEvent = this->RecordGetInitializedEvent_DataOnly<NSLogEvents::SnapshotEventLogEntry, NSLogEvents::EventKind::SnapshotTag>();
         snapEvent->RestoreTimestamp = this->GetLastEventTime();
-        snapEvent->Snap = this->DoSnapshotExtract_Helper((endTime - startTime) / 1000.0);
+
+        JsUtil::BaseHashSet<Js::FunctionBody*, HeapAllocator> liveTopLevelBodies(&HeapAllocator::Instance);
```

Comment:
these few lines are repeated here & above.  Should they be refactored into a method?

Reply:
Personally I didn't see a clean API/call signature for refactoring around this so preferred a bit of duplication to (perhaps) forced code sharing.